src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -69,7 +69,10 @@ public function connect()
             if (isset($options['memory'])) {
                 $dsn = 'sqlite::memory:';
             } else {
-                $dsn = 'sqlite:' . $options['name'] . '.sqlite3';
+                $dsn = 'sqlite:' . $options['name'];
+                if (file_exists($options['name'] . '.sqlite3')) {
```

Comment:
Technically, this should use `is_file`, because `file_exists` will return `true` for a directory as well as a file.